openedx_lti_tool_plugin/views.py

```diff
@@ -155,6 +156,68 @@ def post(self, request: HttpRequest) -> Union[HttpResponseRedirect, LoggedHttpRe
 class LtiToolLaunchView(LtiToolBaseView):
     """LTI 1.3 platform tool launch view."""
 
+    def get_launch_data(self, launch_message: DjangoMessageLaunch) -> dict:
+        """Get LTI launch data from message.
+
+        Args:
+            launch_message: Message object of the launch.
+
+        Returns:
+            A dictionary containing launch data.
+
+        Raises:
+            LtiToolLaunchException when get_launch_data fails.
+        """
+        try:
+            return launch_message.get_launch_data()
+        except LtiException as exc:
+            raise LtiToolLaunchException(_(f'Launch message validation failed: {exc}')) from exc
+
+    def get_identity_claims(
+        self,
+        launch_data: dict
+    ) -> Tuple[str, Union[str, list], str, dict]:
```

Comment:
```suggestion
    ) -> Tuple[str, Union[str, list], str, dict]:
```
What is the return `Union[str, list]` referring to?